attributes/default.rb

```diff
@@ -1,5 +1,19 @@
 default['write_http']['AWS_integration'] = true
-default['write_http']['Ingest_host'] = 'https://ingest.signalfx.com/v1/collectd'
+# We're trying to move away from being locked into the proprietary SignalFx metrics
+# product, and open up options to send our metrics anywhere.
+#
+# To do this we're running OpenTelemetry on Kubernetes that accepts CollectD
+# metrics and exports them to SignalFx, so in the future we could flip where they're
+# exported to.
+#
+# This also means that we only need to manage the SignalFx API key on the
+# Kubernetes deployment of OpenTelemetry rather than keep it up to date on
+# every server
+if node.chef_environment.downcase == "prod"
```

Comment:
I verified with `knife` that prod environment is called `prod` - 

```
(⎈ |infra-eks-thor:test-opentelemetry)aaronkalair@HUD040029 chef_install_configure_collectd % knife node show p-assistedbreakdown-mongo-rs1-use1e-02 -l
Node Name:   p-assistedbreakdown-mongo-rs1-use1e-02
Environment: prod
```

The `downcase` is mostly just paranoia incase it's been typoed as `Prod` or something somewhere.

According to SO this is how you check the environment in Chef - https://serverfault.com/a/417717